### ISSUE TITLE
--diff acts as a starting difficulty with vardiff enabled

### ISSUE THREAD

**Issue by ericecook** [simplecrypto/powerpool#76]
Although I think we should probably not actually rigidly follow the difficulty specified, or a malicious user could flood the server with low difficulty shares, we should probably never adjust to a difficulty lower than what is set. 

_Clarification by icook_: `--diff` argument in password field acts as a starting difficulty on vardiff, but a fixed difficulty on fixed diff. This was not intended behaviour.

**Comment by jakerich** [comment]
I have been Lyra2RE CPU mining (https://github.com/tpruvot/cpuminer-multi) on the variable difficulty port at SimpleMulti.com (stratum+tcp://ca.simplemulti.com:3374). My difficulty gets set to 0.08 after a few seconds and my CPU mines away... However, about every 20 minutes the miner "takes a break" while the stratum sends more work: http://imgur.com/1cmxrEp
![image](https://cloud.githubusercontent.com/assets/11778989/8025758/2c926414-0d16-11e5-96f9-9c34dd415dd7.png). This process takes about 2 to 3 minutes. I would really like to see a steady stream of work flowing in (interestingly, my GPU miners do not suffer from this same stop/start phenomenon on the exact same port).

The solution I have found for this is to point my CPU Miners to the fixed difficulty port (stratum+tcp://ca.simplemulti.com:3373) and set them for the lowest difficulty supported there, "--diff 0.64". This definitely has slowed down my submission rate, but the CPUs are never idle, and the shares are worth significantly more..

**Comment by ericecook** [comment]
@jakerich No, there is no way to manually adjust below 0.64. We've intentionally set the minimum manual difficulty at a moderate level to prevent abuse. SimpleMulti does no throttling, If you were submitting shares too quickly your difficulty would be increased. It is possible there is a bug causing the disconnects (possibly related to https://github.com/simplecrypto/powerpool/issues/113 ?) but this could also be a network connectivity issue.
